<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:#DCDCDC;
border-right:1px solid black;
border-left:1px solid black;
border-top:1px solid black;
border-bottom:1px solid black;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}
QPushButton{
    border:none;
    border-bottom:1px solid black;
    font-size:18px;
    font-weight:700;
 }
QPushButton:hover{border-left:4px solid grey;font-weight:700;}
QCheckBox{
border:none}
</string>
  </property>
  <widget class="QGroupBox" name="DataRecordGroup">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>70</y>
     <width>281</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>数据录入</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="dbGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>数据库状态</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="UserNumberverticalLayout">
      <item>
       <widget class="QLabel" name="UserNumberLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>已存人脸样本数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="UserlcdNumber"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="useinfoGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>261</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>基本信息</string>
    </property>
    <widget class="QPushButton" name="addUserButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>-1</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>增加用户信息</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="ConfirmGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文琥珀</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:none}</string>
     </property>
     <property name="title">
      <string>确认信息</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>29</y>
        <width>241</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="usernumberLabel">
         <property name="font">
          <font>
           <family>华文中宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UsernumlineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="font">
          <font>
           <family>华文中宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UsernamelineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="EnglishLabel">
         <property name="font">
          <font>
           <family>华文中宋</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>英文名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EnglishnamelineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="migrationButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>-1</pointsize>
        <weight>87</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>同步到数据库</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="faceDetectGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>人脸采集</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="OpenVideoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>-1</pointsize>
      <weight>87</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TakephotoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>-1</pointsize>
      <weight>87</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="left_close">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Capturelabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>720</width>
     <height>640</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>摄像头未开启</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="left_visit">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="logGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>201</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>系统日志</string>
   </property>
   <widget class="QTextEdit" name="logtextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="left_mini">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
